build.sh

```diff
@@ -38,8 +38,12 @@ fail() {
   exit 1
 }
 
+# Check for secrets that should not be committed
 /bin/bash ./git-secrets-scan.sh
 
+# Verify dependencies before starting the build
+/bin/bash ./check-dependencies.sh
```

Comment:
rename to `check-prerequisites` (we use the term "dependencies" to represent module/library dependencies)

Reply:
Done